templates/library_combined/Makefile.arm7

```diff
@@ -28,7 +28,7 @@ LIBDIRS		:=
 # ---------------
 
 NAME		:= test7
-BUILDDIR	:= build/arm7
+BUILDDIR	:= build/$(NAME)
```

Comment:
You should probably still preserve the `/arm7` subdirectory here, and in `rom_combined`.

Reply:
Also, if we're doing it here, should I also be appending the ARM9 as well (only inside the `_combined` ones)?

Reply:
I suppose if the ARM7 and ARM9 component always have different `NAME`s, then that's fine. Nevermind.